lib/Controller/MessagesController.php

```diff
@@ -472,7 +472,7 @@ public function mdn(int $id): JSONResponse {
 
 		try {
 			$this->mailTransmission->sendMdn($account, $mailbox, $message);
-			$this->mailManager->flagMessage($account, $mailbox->getName(), $message->getUid(), 'mdnsent', true);
+			$this->mailManager->flagMessage($account, $mailbox->getName(), $message->getUid(), '$mdnsent', true);
```

Comment:
nitpick: I would prefer to have the mdnsent to $mdnsent fix in a separate PR. That keeps the change set small. Additionally we are able to *backport* the flag change to an older stable branch.